ctapipe/calib/camera/calibrator.py

```diff
@@ -178,3 +199,32 @@ def __call__(self, event):
         for telid in tel.keys():
             self._calibrate_dl0(event, telid)
             self._calibrate_dl1(event, telid)
+
+
+def shift_waveforms(waveforms, time_shift_samples):
```

Comment:
May be, for clarity, specify in a comment that the shift is 'towards the left'? (as it should be to correct the mean pixel time to the common mean)

Reply:
yes

Reply:
done